### DISCUSSION TITLE
오늘은 6월 18일 회의했다.

### DISCUSSION THREAD

**Discussion by c-owner** [c-owner/oneuleun_web#16] [회의 로그]
# 오늘은 깃허브

CSS 스타일 

컨셉 : 누가 말 걸어주듯이 ..

​	스텝 단계로 작성

- 메인
  - 사이드 >
    - 테마컬러
      - 배경 애니메이션 처리를 - 영상으로 처리를 
    - 일기장
      - **기능단위** 
        - C (미리보기) R UD
        - 날씨 
        - 기분 - (이모지 리스트 나와서 감정 선택) => 캘린더에 뿌려질거고
        - 위치 - Naver / Kakao / Google MAP
        - 이미지 (파일업로드) 처리 
    - 나의 기분 **[[참고 라이브러리](https://nivo.rocks/circle-packing/)](https://nivo.rocks/circle-packing/)**
      - 일기 감정 [감정 그래프] -> 차트에서 감정 기분변화를 도넛 ()
      - 뇌구조 감정 UI 도입 

- GNB
  - 프로필 >
    - 마이페이지
      - **기능 단위**
        - 이미지업로드 파일(AWS S3 & Naver Cloud - 추후 고민)
        - RUD
    - 로그아웃 

- Member - 유효성 
  - 회원가입 
    - 이메일 발송
  - 로그인 
  - 소셜가입 (ex. kakao, google)
  - 비밀번호 찾기
    - 이메일 발송

---

### 차주 모임 일정까지의 과제

서준이형 - 정보 구조도

태민이형_다솜님  - 스토리보드 + 로고 작업

기훈 - 테이블 설계

**Comment by tjwns3481** [comment]
오늘은, 정보구조도

https://docs.google.com/spreadsheets/d/1HdQ7Cu_wkNQmnd74DB6chAJmNRpeQ3JmRbC6pYk06cY/edit?usp=sharing

**Reply by c-owner** [reply]
고생하셨습니다!👏

**Reply by tjwns3481** [reply]
천만에요 😉
